hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileUpdateHandler.java

```diff
@@ -126,7 +126,7 @@ public void handle(@NonNull final HandleContext handleContext) throws HandleExce
         validateFalse(file.keys() == null, UNAUTHORIZED);
 
         validateMaybeNewMemo(handleContext.attributeValidator(), fileUpdate);
-        validateExpirationTime(fileUpdate, file, handleContext);
+        //        validateExpirationTime(fileUpdate, file, handleContext);
```

Comment:
I had trouble with the same method for the throttle definition tests. I was wondering if it was something to do with the test, apparently not.

Reply:
I fixed it in that PR https://github.com/hashgraph/hedera-services/pull/8253